src/app/public/modules/datepicker/datepicker-config.service.ts

```diff
@@ -5,12 +5,27 @@ import {
 @Injectable()
 export class SkyDatepickerConfigService {
 
+  /**
+   * Specifies the date format for the input.
+   * @default MM/DD/YYYY
```

Comment:
This is a weird one because this class doesn't seem like a true "service" to me, but we're treating it like one here. I wonder if we deliberately designed docs-tools to not look for defaults in services. Let's ask Steve about this one.

Reply:
I pinged Steve with this question since I'd already sent him a couple other items to go over when he's back. I don't think this should hold up the PR though because the defaults for the config properties are both indicated in the related properties on the component that consumes the config.